spring-5-reactive-client/src/main/java/com/baeldung/reactive/simultaneouswebclient/Client.java

```diff
@@ -0,0 +1,66 @@
+package com.baeldung.reactive.simultaneouswebclient;
```

Comment:
Instead lets maybe use com.baeldung.reactive.webclient.simultaneous for the package.